controllers/main.controllers.js

```diff
@@ -11,6 +11,19 @@ exports.getIndex = async (req, res, next) => {
 // @route   POST api/project
 exports.postProject = async (req, res, next) => {
   // Create project
+  let projectObj = {
```

Comment:
1. use try...catch block for error handling
2. Include status codes in response for example: 
`res.status(200).json({ success: true, message: "project created successfully", project });`

Reply:
sure, adding it